HSEARCH-4094 JSR-352 documentation should mention that an additional dependency is necessary (#2435)
hibernate/hibernate-search: documentation/src/main/asciidoc/reference/mapper-orm-indexing-jsr352.asciidoc

== Job Parameters

> The following table will contain all the job parameters we can use to customize the reindexing batch processes.

Reviewer: ```suggestion
The following table contains all the job parameters you can use to customize the mass-indexing job.
```

Not sure this is really necessary, but as you wish.

[[jsr-352-emf-other-implementation]]
=== Other DI-enabled JSR-352 implementations

> If you want to use a different JSR-352 implementation:

Reviewer: ```suggestion
If you want to use a different JSR-352 implementation that happens to allow dependency injection:
```

This is really only necessary if the batch runtime supports DI.
